app/views/disciplines/new.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Create new discipline</h1>
+<%= form_for(@discipline, :html => {class: "form-horizontal", role: "form"}) do |f| %>
```

Comment:
I think the hash here should have spaces inside, e.g. `{ class: "form" }`

Reply:
corrected